app/javascript/components/widgets/forest-change/tree-loss-tsc/index.js

```diff
@@ -20,7 +20,7 @@ export default {
     initial: 'Annual tree cover loss by dominant driver in {location}',
     global: 'Global annual tree cover loss by dominant driver'
   },
-  types: ['global', 'country', 'geostore'],
```

Comment:
I can't verify my assumption because even with the `geostore` present in the array I can't find a custom shape on the main map that would give me a different result from `0 ha`. Please, correct me if I'm wrong but I think that removing the `geostore` from the types array would mess up the analysis in the main map and we only want to hide this widget on the dashboard. So, we'd need to hide that widget only for custom AOIs and only on the dashboards.

Reply:
So, the map layer for this widget is what is driving the need for conditional rendering. The map layers resolution begins to become invalid below zoom level 4. Activate the layer on the map and try zooming in past 4 and you'll notice that the pixels of the layer don't resample. 

If we are disabling this widget for custom AOIs one solution might be to introduce a custom `shouldRender` or similar prop that you can attach to a widget that receives all props after all other filters and you can do what ever you want. Run some turf size function to decide if it should be hidden etc. Somewhere here would work: https://github.com/Vizzuality/gfw/blob/7f192e7e395d6eaf8c8d1605ccdeb379d9bfe48d/app/javascript/components/widgets/selectors.js#L289.

In addition to this @aabdaab is correct that this will affect the map analysis. When the widget isn't rendered on the map, as the layer has an analysisConfig associated with it, on the fly analysis will be triggered to show a single analysis value instead. Is this wanted? We need to get very clear instructions from WRI about when to show this widget and its data. 
1. Is the country pre calc data good enough to show at any level? Or should we manually verify the country size before render
2. Is the pre calc data for AOIs good too? If so then we make sure we don't show this widget until the area is `status: 'saved'`
3. Is it just the showing of the layer next to the widget that is the problem? If so then we can disable the map layer in the dashboard and keep the widget, and not show the widget on the map by changing the `visible` prop of the widgets